sql/src/lib.rs

```diff
@@ -8,18 +8,40 @@
 // CONDITIONS OF ANY KIND, either express or implied. See the License for the
 // specific language governing permissions and limitations under the License.
 
+extern crate ordered_float;
```

Comment:
This happens to be `lib.rs`: the simple case is easy! For many of the query libraries you're right, I didn't do so.

I tend to disagree with 'importing' everything at the top level: it makes it less clear which files depend on which crates. But this is one of Rust's warts: per-module crate dependencies are ugly, and `Cargo.toml` makes the whole thing redundant anyway…